### ISSUE TITLE
Question: Any possibility of making zonkyio optional (along with the xz2 dependency) ?

### ISSUE THREAD

**Issue by helio-frota** [theseus-rs/postgresql-embedded#98]
We are using https://crates.io/crates/rust-lzma and when trying to update postgresql-embedded to 0.13 we can see similar error like this
https://github.com/trustification/trustify/pull/496#issuecomment-2205784223

because `rust-lzma` and `xz2` are linking with the same native library.

thanks

**Comment by brianheineman** [comment]
Hello @helio-frota, thanks for the reporting the issue.  I created new feature flags in #100 to enable `zonky` if desired (it is a non-default feature).

**Comment by helio-frota** [comment]
@brianheineman thanks !

the native-link error gone when using:

```
postgresql_archive = { git = "https://github.com/theseus-rs/postgresql-embedded", rev = "d24fd83d34879fc04a056ecd201e0c7732d527d0" }
postgresql_embedded = { git = "https://github.com/theseus-rs/postgresql-embedded", rev = "d24fd83d34879fc04a056ecd201e0c7732d527d0" }
```

**Comment by brianheineman** [comment]
@helio-frota this change was released in [0.14.0](https://github.com/theseus-rs/postgresql-embedded/releases/tag/v0.14.0)
